### PR TITLE
Filter supported loader versions by game version 

### PR DESCRIPTION
**Pull request by skyrising**
Returns only legacy loader for < 1.12 and loader 0.11.3+ for 1.12

### CODE REVIEW

File: src/main/java/net/fabricmc/meta/utils/LoaderMeta.java [hunk 1]
```diff
@@ -17,21 +18,25 @@
...
 import java.io.FileReader;
 import java.io.IOException;
 import java.net.URL;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class LoaderMeta {
 
 	public static final File BASE_DIR = new File("metadata");
 	public static final String MAVEN_URL = "https://maven.fabricmc.net/";
```
Comment (BluCobalt) [inline src/main/java/net/fabricmc/meta/utils/LoaderMeta.java:38]: what is upstream maven used for here?

Comment (skyrising) [reply src/main/java/net/fabricmc/meta/utils/LoaderMeta.java:38]: For getting available upstream loader versions.

**Review by BluCobalt — APPROVED** [review]

**Review by DeadlyMC — APPROVED** [review]

**Comment by BluCobalt** [comment]
I'll update the hosted meta sometime tomorrow

**Comment by BluCobalt** [comment]
i 100% forgot to do this lol, i'll do it rn

**Pull request merged**